client/my-sites/plugins/marketplace/index.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * External dependencies
+ */
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import { navigation, siteSelection } from 'calypso/my-sites/controller';
+import config from '@automattic/calypso-config';
+import {
+	renderDomainsPage,
+	renderMarketplacePlugin,
+	renderMarketplaceTestPage,
+	renderMarketplaceThankYou,
+	renderPluginsSetupStatusPage,
+} from './controller';
+import { makeLayout, render as clientRender } from 'calypso/controller';
+
+export default function () {
```

Comment:
Now that we're directing an entire section here, we need to think about all the possible routes that this section is responsible for. Essentially, this means intercepting routes like `/marketplace` and/or `/marketplace/:site`.

Reply:
Yeah those routes would potentially include a marketplace library of products in the future, for now it will remain empty. Do you mean that we should forward to a known route if someone tries to visit this route? Not sure how to do that and not sure to where to route to though, open to suggestions.

Reply:
We could do a redirection, or we could display an error screen, we're doing both in Calypso. Whatever works best for the user flows you're building.

Reply:
Will pick this up later seperately c/{cardId}-tr